.changeset/nine-glasses-try.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/control-property-editor': patch
+---
```

Comment:
Please add patch for preview-middleware.

Reply:
Done